my-app/src/.env

```diff
@@ -0,0 +1 @@
+API_BASE_URL=http://localhost
```

Comment:
Great ! Usually, we don't commit this file since this may contain data like:
- Credentials
- Private Keys
- API Urls
- All other things that might introduce a vulnerability, if known